addons/account/models/account_payment.py

```diff
@@ -973,6 +973,9 @@ def unmark_as_sent(self):
     def action_post(self):
         ''' draft -> posted '''
         # Do not allow to post if the account is required but not trusted
+
+        # Refreshing the partner bank is necessary because of its stored computed property. We must make sure the compute triggered once.
+        self.filtered(lambda p: p.require_partner_bank_account and not p.partner_bank_id)._compute_partner_bank_id()
```

Comment:
Can't we solve it by adding a depends? it's weird to call the compute manually

Reply:
Is this preventing the `compute`
https://github.com/odoo/odoo/pull/158133/commits/865c6bd6c6f6e31d1bd003637383daa31007d799#diff-00f59ac6ef8aa991917826a868f8ce891d6f2928662faa07f33adcc665b30b35R805-R808

Could we backport?